File: Java/Assignments/18July_assign/prog1.java
class Main {

	public static void main(String[] args) {

		int a = 10, b = 20;
		boolean ans;
		
		System.out.println("a = " + a + "and b = " + b);
		ans = (a < b);
		System.out.println("a < b = " + ans);
		
		ans = (a > b);
		System.out.println("a > b = " + ans);
		
		ans = (a <= b);
		System.out.println("a <= b = " + ans);
		
		ans = (a >= b);
		System.out.println("a >= b = " + ans);
		
		ans = (a == b);
		System.out.println("a ==  b = " + ans);
		
		ans = (a != b);
		System.out.println("a != b = " + ans);
	}
}

File: Python/27August/prog1.py
print("Enter number :")
num=input()
ans=0
for i  in num:
    n=int(i);
    ans=ans+(n**4)
if(ans==int(num)):
    print("It is Narcisstic Number ");
else:
        print("It is  Not Narcisstic Number ");

